### PR TITLE
Disable SCTP tests for ci-kubernetes-e2e-gce-cos-k8sstable2-alphafeatures

### PR DESCRIPTION
**Pull request by xmudrii**
The following two test suites are failing in `gce-cos-k8sstable2-alphafeatures` on the sig-release-1.19-blocking dashboard for a long time:

* `[sig-network] Networking Granular Checks: Services should function for pod-Service: sctp [Feature:SCTPConnectivity]`
* `[sig-network] Networking should function for pod-pod: sctp [Feature:SCTPConnectivity]`

I've investigated a bit and found that we disabled those tests for other `alphafeatures` jobs. From #17125:

> The tests tagged as [Feature:SCTPConnectivity]" were not supposed to run on Kubernetes CI, however, the regex in the job ci-kubernetes-e2e-gci-gce-alpha-features wasn't excluding it and started to run them, with the problem that they started to fail :)

/assign @justaugustus 
cc: @kubernetes/release-engineering

### CODE REVIEW

File: config/jobs/kubernetes/generated/generated.yaml
Comment (xmudrii) [inline config/jobs/kubernetes/generated/generated.yaml:689]: I'm not sure if this is actually correct. Can someone confirm or point me in the right direction if it's wrong?

Comment (cpanato) [reply config/jobs/kubernetes/generated/generated.yaml:689]: the `Networking-Performance and IPv6` have the `Feature:` as well? why was not like this before?

myabe we keep: `--ginkgo.skip=Networking-Performance|IPv6|Feature:(Volumes| SCTPConnectivity)` ? not sure as well

**Comment by xmudrii** [comment]
/hold
until #22790 is not merged

File: config/jobs/kubernetes/generated/generated.yaml
Comment (xmudrii) [reply config/jobs/kubernetes/generated/generated.yaml]: @cpanato This makes sense to me. I modified the PR to do what you proposed, PTAL again.

**Comment by xmudrii** [comment]
cc: @kubernetes/release-engineering
This is ready for review.

**Review by cpanato — APPROVED** [review]
/lgtm
thanks!

**Review by justaugustus — APPROVED** [review]

**Pull request merged**
